WebUI/Pages/Settings.razor

```diff
@@ -1,7 +1,204 @@
 ﻿@page "/settings"
+@using ConestogaInsidersClub.Data.Models;
+@using ConestogaInsidersClub.Data.DataAccess;
+@using ConestogaInsidersClub.Pages.ViewModels;
+
+@inject IPreferenceService preferenceService;
+@inject IGameService gameService;
+@inject IGameGenreService gameGenreService;
+@inject IUserService userService;
 
 <h1>Settings</h1>
+<br />
+
+@if (user == null)
+{
+    <h3>Loading...</h3>
+}
+else
+{
+    if (editPreferenceMode != true)
+    {
+        <h4>Preferences</h4>
+        <table>
+            <tr>
+                <td><b>Your favourite genre:</b></td>
+                <td><span id="Genre">@(preference.GenreName ?? "*Not set yet*")</span></td>
+            </tr>
+            <tr>
+                <td><b>Your favourite platform:</b></td>
+                <td><span id="Platform">@(preference.Platform ?? "*Not set yet*")</span></td>
+            </tr>
+            <tr>
+                <td><b>Your favourite game:</b></td>
+                <td><span id="Favourite_Game">@(preference.FavouriteGame?.Name ?? "*Not set yet*")</span></td>
+            </tr>
+            <tr>
+                <td><b>Recieve promotional emails</b></td>
+                <td><input disabled type="checkbox" checked="@(preference.ReceivePromotionalEmails ?? false)" id="Recieve_Promotions" /></td>
```

Comment:
I think it would make more sense to see a Yes/No instead of a disabled checkbox
![image](https://user-images.githubusercontent.com/25109066/140666129-c2ea9c72-47b9-419e-9ef3-58c2b7849db6.png)